plugins/filter/jc.py

```diff
@@ -94,7 +94,11 @@ def jc(data, parser, quiet=True, raw=False):
         dictionary or list of dictionaries
 
     Example:
-
+        - name: add the jc prereqs to the running machine
```

Comment:
I think it's more helpful to modify the filter's documentation, and not the comments in the filter's code which are only visible to folks who explicitly look at the filter's source code.

Reply:
But it doesn't, since you didn't edit the documentation. The examples in the documentation are here: https://github.com/ansible-collections/community.general/blob/5216e37983c3d035455fe11c7f25692de8e4ead5/plugins/filter/jc.py#L44-L64